.config/waybar/style.css

```
@@ -0,0 +1,9 @@
+/* Base on the system default style. */
+@import url("file:///etc/xdg/waybar/style.css");
```

Stylelint (17.13.0)

[error] Expected "url("file:///etc/xdg/waybar/style.css")" to be ""file:///etc/xdg/waybar/style.css"" (import-notation)

(import-notation)